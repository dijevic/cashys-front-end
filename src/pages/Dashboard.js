import React from 'react'
import { DashboardComponent } from '../components/dashboard'


export const Dashboard = ({ children }) => {
    return <DashboardComponent>{children}</DashboardComponent>
}
