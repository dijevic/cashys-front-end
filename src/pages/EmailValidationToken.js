import React from 'react'
import { EmailTokenValidationComponent } from '../components/auth/emailValidation'

export const EmailValidationToken = () => {
    return (

        <EmailTokenValidationComponent />

    )
}
