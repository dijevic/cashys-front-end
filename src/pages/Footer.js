import React from 'react'
import { FooterComponent } from '../components/footer'

export const Footer = () => {
    return <FooterComponent />
}