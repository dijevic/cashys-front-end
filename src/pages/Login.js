import React from 'react'
import { LoginStyled } from '../components/auth/login'







export const Login = () => {
    return (
        <LoginStyled />
    )
}
