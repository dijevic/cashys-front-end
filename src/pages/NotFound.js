

import React from 'react'

export const NotFound = () => {
    return (
        <p>Page not found 404 X-X</p>
    )
}
