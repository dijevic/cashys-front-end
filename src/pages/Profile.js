import React from 'react'



export const Profile = () => {

    return <h1>PROFILE</h1>
}