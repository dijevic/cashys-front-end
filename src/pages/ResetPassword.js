import React from 'react'
import { ResetPasswordComponent } from '../components/auth/resetpassword/ResetPassword'

export const ResetPassword = () => {
    return <ResetPasswordComponent />
}
