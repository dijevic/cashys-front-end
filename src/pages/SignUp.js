import React from 'react'
import { Login } from './Login'


export const SignUp = () => {


    return (
        <Login></Login>
    )
}