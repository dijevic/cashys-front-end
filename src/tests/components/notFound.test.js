import { shallow } from "enzyme/build"
import { NotFound } from "../../pages/NotFound"

describe('test the 404 not found route', () => {

    const wrapper = shallow(<NotFound />)

    test('should match the snapshot', () => {

        expect(wrapper).toMatchSnapshot()

    })
    test('should get the text NOT FOUND', () => {


        const p = wrapper.find('p')

        expect(p.text()).toBe('Page not found 404 X-X')


    })
})