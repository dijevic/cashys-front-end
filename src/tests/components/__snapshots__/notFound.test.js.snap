// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test the 404 not found route should match the snapshot 1`] = `
<p>
  Page not found 404 X-X
</p>
`;
